Libs/Widgets/ctkAxesWidget.cpp

```diff
@@ -258,7 +258,7 @@ void ctkAxesWidget::paintEvent(QPaintEvent *)
     {
     //if (d->HighlightAxes)
       {
-      QPen pen;
+      QPen pen(this->palette().color(QPalette::Active, QPalette::WindowText));
```

Comment:
`QPen pen(this->palette().color(this->isEnabled() ? QPalette::Active : QPalette::Inactive, QPalette::WindowText));`